src/OrchardCore.Modules/OrchardCore.Github/Recipes/TwitterSigninSettingsStep.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Threading.Tasks;
+using OrchardCore.Github.Services;
+using OrchardCore.Github.Settings;
+using OrchardCore.Recipes.Models;
+using OrchardCore.Recipes.Services;
+
+namespace OrchardCore.Github.Recipes
+{
+    /// <summary>
+    /// This recipe step sets Microsoft Account settings.
+    /// </summary>
+    public class GithubSigninSettingsStep : IRecipeStepHandler
+    {
+        private readonly IGithubAuthenticationService _twitterLoginService;
```

Comment:
Variable name does not match Interface and class ```<summary>``` is also incorrect.

Reply:
Also the class should be changed to GithubSigninSettingsStep.cs

Reply:
thanks @Icidis , the PR is already merged, so you can open a new one if you want to contribute, or tell me to do it.